packages/web/components/Form/SwitchField.js

```diff
@@ -80,11 +85,13 @@ const SwitchField = ({ label, form, name, help, validation, ...checkboxProps })
 				{...checkboxProps}
 			/>
 			<Row>
-				<SwitchLabel htmlFor={name} checked={isChecked}>
-					<p>{isChecked ? 'Sim' : 'Não'}</p>
-					<span />
-				</SwitchLabel>
-				{help && <Help id={name} HelpComponent={help} />}
+				<SwitchLabelWrapper>
+					<SwitchLabel htmlFor={name} checked={isChecked}>
+						<p>{isChecked ? 'Sim' : 'Não'}</p>
```

Comment:
Esse aqui já temos nos locales:
```suggestion
						<p>{isChecked ? t('common:yes') : t('common:no')}</p>
```

Reply:
Fica bem melhor com o locale mesmo mas tenho pouca experiência com testes e não consegui encontrar uma forma de fazer um mock que renderize as traduções.

Reply:
@calaca Nos testes as traduções estão mockadas para renderizar o slug. Não precisa renderizar a tradução em si :).

Reply:
@nicholasio ah sim! Perfeito! Vou fazer esse ajuste que o Mateus pediu e "finalizar" sem a integração do ConextAPI por agora só pra não segurar mais esse PR. Quando essa integração foi feita ajusto o componente Price também.